react/features/subtitles/reducer.ts

```diff
@@ -70,6 +75,11 @@ ReducerRegistry.register<ISubtitlesState>('features/subtitles', (
  */
 function _removeTranscriptMessage(state: ISubtitlesState, { transcriptMessageID }: { transcriptMessageID: string; }) {
     const newTranscriptMessages = new Map(state._transcriptMessages);
+    const message = newTranscriptMessages.get(transcriptMessageID);
+
+    if (message && !message.final) {
+        state._cachedTranscriptMessages.set(transcriptMessageID, message);
```

Comment:
Technically at the moment there isn't a need for this because we don't use this map in any subscribers, so we don't care if redux will notify or not notify anybody about this change. 

But I totally agree it is a good practice and I guess it can save a lot of time for the next person who actually wants to use it in subscriber :)

Will change it!